solc/CommandLineParser.cpp

```diff
@@ -739,6 +741,10 @@ General Information)").c_str(),
 
 	po::options_description metadataOptions("Metadata Options");
 	metadataOptions.add_options()
+		(
+			g_strNoAppendMetadata.c_str(),
+			"Do not append metadata to the end of the bytecode."
```

Comment:
"Metadata" is somewhat ambiguous in this context. We have metadata at the end of the binary and then metadata stored in JSON. How about we invent a name for the binary one to be able to refer to it more precisely? For example what do you think about something like "metadata anchor". Option naming would also be easier - we could then call it `--no-metadata-anchor`.

```suggestion
			"Do not append metadata anchor at the end of the bytecode. The metadata anchor is a small bit of CBOR data that identifiers the compiler version and may include a hash of the full metadata."
```

If we go with this, the descriptions of the two options below should also be updated to match.

Reply:
I'm not sure about anchor here. It's a new terminology :)

Reply:
I mean, would be great to have any clear term for this. Calling both metadata is not helpful.

We just had a discussion with @ekpyron today about how ambigous dynamically-sized and statically-sized really are...